spec/integration_spec.rb

```diff
@@ -109,6 +109,32 @@ def pptx(pptx_name)
     end
   end
 
+  context "with a slide with text" do
+    let(:theme) { OpenXml::Pptx::Parts::Theme.new }
+    let(:master) { OpenXml::Pptx::Parts::SlideMaster.new(theme) }
+    let(:layout) { OpenXml::Pptx::Parts::SlideLayout.new(master) }
+    let(:slide) { OpenXml::Pptx::Parts::Slide.new(layout) }
+
+    before do
+      slide.add_text("Hello World")
```

Comment:
I'm not sure. I don't much care for the `<<` as I don't believe it shows intention. As of right now we only have text, which ultimately is an instance of `Text`. I can change the name of that to `TextBox` if you like.  Ultimately what you are doing looks very much like the xml and I was trying to avoid the user just recreating the xml with a different look. The issue is that when you add a text box more than just `t > p > r` are added. There are three separate pieces of xml that are added and needed to place a box.  What I called `Text` is the grouping of all three of these xml blocks that it takes to place the text on a slide.

Is it okay to extract the shape and complexity of the xml away from the user? I am not sure at this time. I hope that it is because that seems like one complicated piece. I do expect that it will change to be more like `slide.add_text(text_object)` and the text object is what I'm creating in `add_text` at this time. That is because in doing this I was defaulting all the different properties to get text in a certain location. We will need that to be configureable by the end user according to our conversation.

Here is the `Text` object I'm talking about: https://github.com/openxml/openxml-pptx/pull/24/files#diff-0480615b04b4b802d23af72099d7d332R1